Dockerfile

```diff
@@ -129,9 +132,8 @@ ENV MS_DEBUGLEVEL=0 \
     IO_TIMEOUT=40 \
     GET_ENV=env
 
-RUN adduser www-data root \
-    && chmod -R g+w ${APACHE_CONFDIR} ${APACHE_RUN_DIR} ${APACHE_LOCK_DIR} ${APACHE_LOG_DIR} /etc/mapserver /var/lib/apache2/fcgid /var/log \
```

Comment:
The log dir is not more writable ...

Reply:
it is because created above with `-m 777`